iglu-registry/README.md

```diff
@@ -0,0 +1,203 @@
+# Iglu Registry
+
+This folder contains
+
+* docker images of [Iglu Registry](iglu-registry)
+  * `0.3.0`
+* a docker compose example to run [Iglu Registry](iglu-registry) with [Postgres](https://github.com/docker-library/postgres)
+
+## Introduction
+
+[Iglu Registry](iglu-registry) images are based on [Snowplow base image][base-image] which leverages [the Java 8 Alpine image][alpine-image].
+
+The Iglu Registry runs under [dumb-init][dumb-init] which handles reaping zombie processes
+and forwards signals on to all processes running in the container. This image also uses
+[su-exec][su-exec], as a sudo replacement, to run the Iglu Registry as the non-root `snowplow` user.
+
+The container exposes the `/snowplow/config` volume to store the registry configuration. If this
+folder is bind mounted then ownership will be changed to the `snowplow` user.
+
+The `-XX:+UnlockExperimentalVMOptions` and `-XX:+UseCGroupMemoryLimitForHeap` JVM options will be
+automatically provided when launching the registry in order to make the JVM adhere to the memory
+limits imposed by Docker. For more information, see [this article][jvm-docker-article].
+
+Additional JVM options can be set through the `SP_JAVA_OPTS` environment variable.
+
+### Build
+
+1) Clone repository
+
+    `$ git clone git@github.com:snowplow/snowplow-docker.git`
+
+2) Navigate to `iglu-registry/`
+
+    `$ cd iglu-registry/`
+
+3) Build image    
+
+    `$ docker build -t iglu-registry:0.x.0 iglu-registry/0.x.0`
```

Comment:
Right command here is `docker build -t iglu-registry:0.x.0 0.x.0`

Reply:
Ah, right. Will update.